Sources/StreamChat/Config/ChatClientConfig.swift

```diff
@@ -64,6 +64,7 @@ public struct ChatClientConfig {
     /// but has to be initiated manually by calling `connect`.
     ///
     /// Is `true` by default.
+    @available(*, deprecated, message: "Has no effect anymore. It now always behaves like it was set to `true`.")
     public var shouldConnectAutomatically = true
```

Comment:
message: "This flag has no effect anymore. The flow for setting and for connecting the user has been unified to the `connectUser` set of methods."